MycroftAI/mycroft-core
# Use get_wch() to support unicode input

## Description
Replace getch with get_wch to accept unicode characters from the command line interface.

## How to test
- Check that normal ascii inputs still works.
- make sure `speak Åke är bäst¡!¡!` returns the same text as entered

## Contributor license agreement signed?
CLA [Yes]
